src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -484,7 +484,7 @@ class TaskStatusBarItem extends Themable implements IStatusbarItem {
 		element.appendChild(labelElement);
 
 		let label = new OcticonLabel(labelElement);
-		label.title = nls.localize('runningTasks', "Show Running Tasks");
+		label.title = nls.localize('runningTasks', "Show Running Task Outputs");
```

Comment:
IMO adding outputs here makes it more confusing since it might be mistaken for the Output view. Why do you suggest to add this.